run.py

```diff
@@ -1,66 +1,61 @@
-from typing import List
+from typing import List, Optional
 
 from blocks.pipeline import Pipeline
 from configs.constants import Const
 from library.evaluation import calibration_metrics, classification_metrics
-from library.examples.hate_speech import (
-    cross_dataset_run_configs,
-    ensemble_pipeline_hf,
-    huggingface_baseline,
-    preprocess_config,
-    tweeteval_hate_speech_run_configs,
-    vader,
-)
+from library.examples.hate_speech import (cross_dataset_run_configs,
+                                          ensemble_pipeline_hf,
+                                          huggingface_baseline,
+                                          preprocess_config,
+                                          tweeteval_hate_speech_run_configs,
+                                          vader)
 from plugins import WandbConfig, WandbPlugin
 from runner.runner import Runner
-from type import Evaluators, PreprocessConfig, RunConfig
+from type import Evaluators, Experiment, PreprocessConfig
 
 
 def run(
-    pipeline: Pipeline,
-    preprocess_config: PreprocessConfig,
-    project_id: str,
-    run_configs: List[RunConfig],
-    metrics: Evaluators,
+    experiments: List[Experiment],
+    save_remote: Optional[
+        bool
+    ] = None,  # If set True all models will try uploading (if configured), if set False it overwrites uploading of any models (even if configured)
+    remote_logging: Optional[
```

Comment:
it looks like remote_logging was not used anywhere! did we have it wired up on latest `main`?

Reply:
It is used in one place: in ``run.py``! 

```python
        logger_plugins = (
            [
                WandbPlugin(
                    WandbConfig(
                        project_id=project_id,
                        run_name=config.run_name + "-" + pipeline.id,
                        train=True,
                    ),
                    dict(
                        run_config=config.get_configs(),
                        preprocess_config=preprocess_config.get_configs(),
                        pipeline_configs=pipeline.get_configs(),
                    ),
                )
            ]
            if config.remote_logging
            else []
        )```

Reply:
sorry, my mistake! I meant save_remote, not remote_logging!